charts/tekton-operator/templates/kubernetes-crds.yaml

```diff
@@ -2,6 +2,46 @@
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  name: manualapprovalgates.operator.tekton.dev
+  labels:
+    version: v0.71.0
+    operator.tekton.dev/release: v0.71.0
```

Comment:
@piyush-garg in other places we use the version as `v0.68.0` 
if we use the same version, it will be easier to replace with `sed` command on release time.

Reply:
Yup I also thought about it, but as manual approval was shipped first in 0.71.0, so decided not to put 0.68.0 as it may confuse

Reply:
I have update for all version values to keep consistency